Plots/Polygons/NCL_polyg_18.py

```diff
@@ -0,0 +1,133 @@
+"""
+NCL_polyg_18.py
+==============
+This script illustrates the following concepts:
+   - Adding lines, markers, and polygons to a map
+   - Drawing lines, markers, polygons, and text in inset axes
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/polyg_18.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/polyg_18_2_lg.png
+"""
+
+###############################################################################
+# Import packages:
+# ----------------
+import cartopy.crs as ccrs
+import matplotlib.pyplot as plt
+import cartopy
+from geocat.viz import util as gvutil
+import matplotlib.patches as mpatches
+from cartopy.mpl.ticker import LatitudeFormatter, LongitudeFormatter
+import numpy as np
+###############################################################################
```

Comment:
Consider reorganizing the order of import statements. Imports from the same package should be grouped together. It's also a good idea to separate imports from geocat from the other imports with a new line to call attention to packages that aren't as common and may need to be installed.